config/locales/shared/en.yml

```diff
@@ -8,10 +8,9 @@ en:
       gov_heading: Official websites use .gov
       how: Here’s how you know
       landmark_label: Official government website
-      lock: Lock
       lock_description: A locked padlock
       official_site: An official website of the United States government
-      secure_description_html: A <strong>lock</strong> (%{lock_icon}) or
+      secure_description_html: A <strong>lock</strong> ( %{lock_icon} ) or
```

Comment:
The inclusion of extra spaces here is intentional, because the removal of the extra `span` wrapper results in collapsed spacing, since we were indirectly relying on prefixing spaces ahead of the `<svg>` tag:

https://github.com/18F/identity-idp/blob/f660bfe605586346dc2705becb64b706fb751178/app/views/shared/_banner-lock-icon.html.erb#L2

Since the partial now has its spacing collapsed, I moved to embed them in the string itself, which I think does a better job of accurately reflecting the expectation of a space around the icon.